src/vue-timepicker.vue

```diff
@@ -10,6 +10,7 @@ export default {
   name: 'VueTimepicker',
 
   props: {
+    inputClass: {type: String},
```

Comment:
Hi. I think adding `default: 'display-time'` is better. Because the `display-time` class is used for applying the default style.